doc/crypto/api/ops/signature.rst

```diff
@@ -56,6 +56,7 @@ There are three categories of asymmetric signature algorithm in the |API|:
     The following algorithms are in this category:
 
     | `PSA_ALG_PURE_EDDSA`
+      `PSA_ALG_EDDSA_CTX`
```

Comment:
This needs a leading '|' character to treat this as a paragraph of left-justified short lines